osbs-client.spec

```diff
@@ -175,27 +177,11 @@ LANG=en_US.utf8 py.test-%{python2_version} -vv tests
 %files
 %doc README.md
 %{_bindir}/osbs
-
-
-%files -n python-osbs-client
-%doc README.md
 %{!?_licensedir:%global license %doc}
 %license LICENSE
-%{_bindir}/osbs-%{python2_version}
-%{_bindir}/osbs-2
-%{python2_sitelib}/osbs*
-%dir %{_datadir}/osbs
-%{_datadir}/osbs/*.json
-%ghost %config(noreplace) %{_datadir}/osbs/orchestrator_customize.json
-%ghost %config(noreplace) %{_datadir}/osbs/prod_customize.json
-%ghost %config(noreplace) %{_datadir}/osbs/worker_customize.json
-
 
 %if 0%{?with_python3}
 %files -n python3-osbs-client
-%doc README.md
-%{!?_licensedir:%global license %doc}
-%license LICENSE
```

Comment:
Shouldn't all the subpackages ship the LICENSE file?

Reply:
yeah will change it